app/gui2/src/stores/graph/graphDatabase.ts

```diff
@@ -11,13 +11,16 @@ import * as set from 'lib0/set'
 import {
   IdMap,
   visMetadataEquals,
+  type ContentRange,
   type ExprId,
   type NodeMetadata,
   type VisualizationMetadata,
 } from 'shared/yjsModel'
 import { ref, type Ref } from 'vue'
+import type { Import } from '../imports'
 
 export class GraphDb {
+  imports: Ref<{ import: Import; span: ContentRange }[]> = ref([])
```

Comment:
`imports` are not used by anything in GraphDb and I think they should be separate property in Graph store.